testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddKeyStoreOnlineTest.java

```diff
@@ -0,0 +1,120 @@
+package org.wildfly.extras.creaper.commands.elytron.tls;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.commands.elytron.AbstractElytronOnlineTest;
+import org.wildfly.extras.creaper.commands.security.realms.AddPropertiesAuthentication;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+
+@RunWith(Arquillian.class)
+public class AddKeyStoreOnlineTest extends AbstractElytronOnlineTest{
```

Comment:
Change `... extends AbstractElytronOnlineTest{` to `... extends AbstractElytronOnlineTest {`